### PR TITLE
Fix units

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

- [x] fixed wrong unit for `Physics::P_GROUND`
- [x] add units to constants
- [x] fixed drag calculation (wrong units and missing `vNorm`)
- [x] adapted tests
- [ ] Test 1 month run with LADDS

### CODE REVIEW

**Comment by FG-TUM** [comment]
I guess that also means we should change all example data's bc_inf from 2e-5 to 2e-3. That way we actually get a `c_D` of 2.5

**Comment by gomezzz** [comment]
> I guess that also means we should change all example data's bc_inf from 2e-5 to 2e-3. That way we actually get a c_D of 2.5

Why 2.5 though? Aren't we aiming for 2.2?

**Review by gomezzz — CHANGES_REQUESTED** [review]
Could you run a quick  1month run or something just as a sanity check that we don't rapidly decay everything now? Because that certainly wouldn't be right either. (more indepth analysis of stability of orbital elements should probably follow later).

Also, tests are failing :)

**Comment by FG-TUM** [comment]
> Why 2.5 though? Aren't we aiming for 2.2?

We - yes. But apparently this repo in it's example input not.

File: src/satellitePropagator/physics/Constants.h [hunk 1]
```diff
@@ -28,21 +28,21 @@ inline constexpr double RAD_FACTOR = M_PIl / 180;
  */
 inline constexpr double GM_EARTH = 3.986004407799724e+5;
```
[the remark this answers is not in the document]
Comment (FG-TUM) [reply src/satellitePropagator/physics/Constants.h:29]: so you want this to be renamed to `MU_EARTH`? And I guess accordingly for the other bodies?

File: src/satellitePropagator/physics/Constants.h [hunk 2]
```diff
@@ -160,24 +160,24 @@ inline constexpr double P_SRP = 4.56e-3;
  */
 inline constexpr double OMEGA = 282.94;
 /**
- * @brief Atmospheric scale height.
+ * @brief Atmospheric scale height [km].
  *
  * Used in:
  * - Acceleration::SolComponent
```
[the remark this answers is not in the document]
Comment (FG-TUM) [reply src/satellitePropagator/physics/Constants.h]: it actually isn't. fixed -> a743fd9

File: src/satellitePropagator/physics/Constants.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (gomezzz) [reply src/satellitePropagator/physics/Constants.h:29]: Meh, then let's keep it, greek letters don't make for great code anyways

**Review by gomezzz — APPROVED** [review]
Looks good, only thing as comment in earlier review:
> Could you run a quick 1month run or something just as a sanity check that we don't rapidly decay everything now? Because that certainly wouldn't be right either. (more indepth analysis of stability of orbital elements should probably follow later).

Maybe we create an issue for this in the LADDS repo?

File: src/satellitePropagator/physics/Constants.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (FG-TUM) [reply src/satellitePropagator/physics/Constants.h:29]: > Maybe we create an issue for this in the LADDS repo?

No I want to fix test this before we merge this.

**Comment by FG-TUM** [comment]
> Maybe we create an issue for this in the LADDS repo?

No I want to fix test this before we merge this.

**Review by gomezzz — APPROVED** [review]

**Pull request merged**